test/unit/published_edition_presenter_test.rb

```diff
@@ -74,5 +74,13 @@ class PublishedEditionPresenterTest < ActiveSupport::TestCase
       output = presenter.render_for_publishing_api(republish: false)
       assert_equal 'major', output[:update_type]
     end
+
+    should 'use updated_at value if public_updated_at is nil' do
+      updated_at = 1.minute.ago
+      edition = FactoryGirl.create(:edition, state: "draft", updated_at: updated_at)
+      assert_nil edition.public_updated_at
+      output = PublishedEditionPresenter.new(edition).render_for_publishing_api
+      assert_equal edition.updated_at, output[:public_updated_at]
+    end
```

Comment:
To clarify what I meant: it seems worth adding a line to validate a draft edition against the schema, because there may be other things that are invalid about draft editions. We're already doing it for published editions.